pkg/mount/mount.go

```diff
@@ -116,9 +116,7 @@ func RecursiveUnmount(target string) error {
 			continue
 		}
 		if err := Unmount(m.Mountpoint); err != nil && i == len(mounts)-1 {
-			if mounted, err := Mounted(m.Mountpoint); err != nil || mounted {
-				return err
-			}
+			return err
```

Comment:
We do not want to bubble errors when some other tool unmounted the file system.

Reply:
same here, `Mount()` ignores EINVAL